FTBFS xorg-xserver:

### Problem

Discovered when running Xvfb 

```console
19e241998786:/work/packages#  Xvfb :0 -screen 0 1024x768x24
Xvfb: error while loading shared libraries: libgallium-24.2.5.so: cannot open shared object file: No such file or directory
19e241998786:/work/packages# apk search so:libgallium-24.2.5.so
mesa-libgallium-24.2.5-r0
```

### Build logs

```console
2024/10/18 13:49:11 INFO libtool: link: x86_64-pc-linux-gnu-gcc -DHAVE_DIX_CONFIG_H -DHAVE_DIX_CONFIG_H -Wall -Wpointer-arith -Wmissing-declarations -Wformat=2 -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Wbad-function-cast -Wold-style-definition -Wdeclaration-after-statement -Wunused -Wuninitialized -Wshadow -Wmissing-noreturn -Wmissing-format-attribute -Wredundant-decls -Wlogical-op -Werror=implicit -Werror=nonnull -Werror=init-self -Werror=main -Werror=missing-braces -Werror=sequence-point -Werror=return-type -Werror=trigraphs -Werror=array-bounds -Werror=write-strings -Werror=address -Werror=int-to-pointer-cast -Werror=pointer-to-int-cast -fno-strict-aliasing -fno-strict-aliasing -D_DEFAULT_SOURCE -D_BSD_SOURCE -DHAS_FCHOWN -DHAS_STICKY_DIR_BIT -I/usr/include/X11/dri -I/usr/include/libdrm -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I../../include -I../../include -I../../Xext -I../../composite -I../../damageext -I../../xfixes -I../../Xi -I../../mi -I../../miext/sync -I../../miext/shadow -I../../miext/damage -I../../render -I../../randr -I../../fb -I../../dbe -I../../present -fvisibility=hidden -O2 -Wall -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -march=x86-64-v2 -mtune=broadwell -Wl,--as-needed -Wl,-O1 -Wl,--sort-common -Wl,-z -Wl,relro -Wl,-z -Wl,now -Wl,-z -Wl,noexecstack -o Xvfb InitInput.o InitOutput.o miinitext.o -Wl,--export-dynamic  ../../fb/.libs/libfb.a ../../xfixes/.libs/libxfixes.a ../../Xext/.libs/libXext.a ../../dbe/.libs/libdbe.a ../../record/.libs/librecord.a ../../glx/.libs/libglx.a ../../glx/.libs/libglxvnd.a ../../randr/.libs/librandr.a ../../render/.libs/librender.a ../../damageext/.libs/libdamageext.a ../../dri3/.libs/libdri3.a ../../present/.libs/libpresent.a ../../miext/sync/.libs/libsync.a ../../miext/damage/.libs/libdamage.a ../../miext/shadow/.libs/libshadow.a ../../Xi/.libs/libXi.a ../../xkb/.libs/libxkb.a ../../xkb/.libs/libxkbstubs.a ../../composite/.libs/libcomposite.a ../../dix/.libs/libmain.a ../../dix/.libs/libdix.a ../../mi/.libs/libmi.a ../../os/.libs/libos.a -lcrypto ../../Xi/.libs/libXistubs.a -lGL -lpixman-1 -lXfont2 -lXau -lxshmfence -lXdmcp -lm
2024/10/18 13:49:11 WARN /usr/lib/gcc/x86_64-pc-linux-gnu/14/../../../../x86_64-pc-linux-gnu/bin/ld: warning: libgallium-24.2.5.so, needed by /usr/lib/gcc/x86_64-pc-linux-gnu/14/../../../../lib64/libGL.so, not found (try using -rpath or -rpath-link)
2024/10/18 13:49:12 WARN /usr/lib/gcc/x86_64-pc-linux-gnu/14/../../../../x86_64-pc-linux-gnu/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/14/../../../../lib64/libGL.so: undefined reference to `dri_loader_get_extensions@libgallium-24.2.5.so'
2024/10/18 13:49:12 WARN collect2: error: ld returned 1 exit status
2024/10/18 13:49:12 WARN make[3]: *** [Makefile:668: Xvfb] Error 1
2024/10/18 13:49:12 INFO make[3]: Leaving directory '/home/build/hw/vfb'
2024/10/18 13:49:12 WARN make[2]: *** [Makefile:734: all-recursive] Error 1
2024/10/18 13:49:12 INFO make[2]: Leaving directory '/home/build/hw/vfb'
2024/10/18 13:49:12 INFO make[1]: Leaving directory '/home/build/hw'
2024/10/18 13:49:12 WARN make[1]: *** [Makefile:582: all-recursive] Error 1
2024/10/18 13:49:12 WARN make: *** [Makefile:791: all-recursive] Error 1
2024/10/18 13:49:12 INFO make: Leaving directory '/home/build'
2024/10/18 13:49:12 ERRO ERROR: failed to build package. the build environment has been preserved:
2024/10/18 13:49:12 INFO   workspace dir: /tmp/melange-workspace-2230471859
2024/10/18 13:49:12 INFO   guest dir: /tmp/melange-guest-85810594
2024/10/18 13:49:12 ERRO failed to build package: unable to run package xorg-server pipeline: unable to run pipeline: unable to run pipeline: exit status 2
make[1]: *** [Makefile:114: packages/x86_64/xorg-server-21.1.13-r1.apk] Error 1
make[1]: Leaving directory '/home/user/src/packages/wolfi-os'
make: *** [Makefile:104: package/xorg-server] Error 2
```

--- a/xorg-server.yaml
+++ b/xorg-server.yaml
@@ -1,7 +1,7 @@
 package:
   name: xorg-server
   version: 21.1.13
-  epoch: 1
+  epoch: 2
   description: "X Server"
   copyright:
     - license: SGI-B-2.0
@@ -10,6 +10,7 @@ package:
       - dbus
       - dbus-x11
       - font-misc
+      - mesa-libgallium
       - systemd
       - ttf-dejavu
       - xkbcomp
@@ -41,6 +42,7 @@ environment:
       - mesa-dev
       - mesa-gbm
       - mesa-gl
+      - mesa-libgallium
       - openssl-dev
       - pixman-dev
       - zlib-dev
@@ -113,7 +115,7 @@ subpackages:
         with:
           repository: https://salsa.debian.org/xorg-team/xserver/xorg-server
           branch: debian-unstable
-          expected-commit: d05d14f560f009df48c20f6dcc6a4428ed426f0a
+          expected-commit: ecfb1ba65b2a7ec9a999954202311d1f27f635d2
       - working-directory: debian/local
         pipeline:
           - runs: |

